app/models/cdr/table.rb

```diff
@@ -12,11 +12,18 @@
 #
 
 class Cdr::Table < Cdr::Base
-  extend PgCdrPartitioning
-
   self.table_name = 'sys.cdr_tables'
 
-  has_paper_trail class_name: 'AuditLogItem'
+  include PgPartitioningMixin
+
+  self.partitioned_model = Cdr::Cdr
+  self.partition_schema = 'cdr'.freeze
```

Comment:
Give default value to if, if not defined. Take the same schema as `partitioned_model` is.